opusfilter/filters.py

```diff
@@ -182,9 +183,18 @@ def accept(self, score):
 class LanguageIDFilter(FilterABC):
     """Language identification confidence filter"""
 
-    def __init__(self, languages=None, id_method='langid', thresholds=None, **kwargs):
+    def __init__(self, languages=None, id_method='langid', thresholds=None, 
```

Comment:
There's some trailing whitespace here and several other places - please see and fix the flake8 warning from the automatic checks. (To run the same manually, just install flake8 and write `flake8 opusfilter/` in the project root.)

Reply:
Ok. We should add this Flake8 and pytest Howto in the main readme in a "How to contribute" section. Would help people to contribute :)

Reply:
Sure, that's a good point. I have now added [such a section to the README](https://github.com/Helsinki-NLP/OpusFilter/blob/develop/README.md#how-to-contribute).